card/db/migrate/20150220141031_create_deck_routes_file.rb

```diff
@@ -0,0 +1,14 @@
+# -*- encoding : utf-8 -*-
+
+class CreateDeckRoutesFile < ActiveRecord::Migration
+  def up
+    routes_path = File.join Wagn.root, 'config', 'routes.rb'
+    if !File.exist? routes_path
+      template_path = File.join Wagn.gem_root, 'lib', 'wagn', 'generators', 'wagn', 'templates', 'config', 'routes.erb'
+      @include_jasmine_engine = false
+      decko_path = ''
+      routes = ERB.new(File.read template_path).result binding
+      File.open(routes_path, "w") { |f|  f.write routes }
+    end
+  end
+end
```

Comment:
My previous comment explains how app decks handle it, right?  The application.rb can override the default setting to get the routes file from the gem.

Engines, obviously, don't have an application.rb.  They have an engine.rb.  There you can override the default setting to get the routes file from the deck.